object.js

```diff
@@ -69,7 +70,8 @@ const tonicHTML = `
         <li class="tag">${tonic.technologies[2]}</li>
       </ul>
       <div class="ref">
-        <button>See project</button>
+        <button>See live</button>
+        <button class="fastt">See source</button>
```

Comment:
- [ ] The rendering of your popup on the desktop is the same design on the mobile, which does not fit the project requirements. From the comparison below, you can see it's far from what's asked on the figma template. 
- [ ] Just as I made mentioned on the mobile, also make sure each of the cards has a popup by using a dynamic render code to list each from your array table. a `map` will come in handy here 👍 
- The aim is to stick to user requirements, and also give a user a better experince 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50990925/224521418-e4023281-5ebc-47c2-b56a-40e85eb6da2d.png)

## Figma:
![image](https://user-images.githubusercontent.com/50990925/224521446-537885ad-397f-4ea3-8034-aa993236fc4a.png)